src/plugins/tagfilter/_base.scss

```diff
@@ -0,0 +1,25 @@
+/**
+ * @title Tag filter CSS
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @duboisp
+ *
+ * Note: Depend also on the CSS for the filter WET plugin
+ */
+
+.wb-tagfilter.provisional {
+	.wb-tgfltr-out {
```

Comment:
Side note:
Originally the intend was to also extend the filter plugin to display persistent faded text when text was hidden. We can look at that later on.